lib/supply/client.rb

```diff
@@ -18,38 +19,27 @@ class Client
     # @!group Login
     #####################################################
 
-    # Initializes the auth_client and api_client using the specified information
-    # @param path_to_key: The path to your p12 file
-    # @param issuer: Email addresss for oauth
-    # @param passphrase: Passphrase for the p12 file
-    def initialize(path_to_key: nil, issuer: nil, passphrase: nil)
-      passphrase ||= "notasecret"
+    # Initializes the android_publisher and its auth_client using the specified information
+    # If not specified, it tries to take the value of the secrets.json from ENV['GOOGLE_APPLICATION_CREDENTIALS']
+    # @param path_to_service_account_json: The path to your service account Json file
+    def initialize(path_to_service_account_json: nil)
+      # REVIEW: do we want to force override ? If not, we don't get the 'proper'
+      # printed table output
+      ENV['GOOGLE_APPLICATION_CREDENTIALS'] = File.expand_path path_to_service_account_json if path_to_service_account_json
```

Comment:
How about:
```
ENV['GOOGLE_APPLICATION_CREDENTIALS'] ||= File.expand_path(path_to_service_account_json) if path_to_service_account_json
```
And then maybe a `UI.message` with something like "Using GOOGLE_APPLICATION_CREDENTIALS from ..." so that the user can see what the end result in that variable is, without us having to force a value into it.